.ddqa/config.toml

```diff
@@ -78,6 +78,28 @@ jira_statuses = [
 github_team = "network-device-monitoring"
 github_labels = ["team/network-device-monitoring"]
 
+[teams."Network Device Monitoring Core"]
```

Comment:
do we need another one for ndm integrations?

Reply:
Oh definitely, this was supposed the block right under but looks like I did not commit that change!